features-json/u2f.json

```diff
@@ -264,15 +280,15 @@
       "95":"y #3",
       "96":"y #3",
       "97":"y #3",
-      "98":"y #3",
-      "99":"y #3",
-      "100":"y #3",
-      "101":"y #3",
-      "102":"y #3",
-      "103":"y #3",
-      "104":"y #3",
-      "105":"y #3",
-      "106":"y #3"
+      "98":"d #3",
+      "99":"d #3",
+      "100":"d #3",
+      "101":"d #3",
+      "102":"d #3",
+      "103":"d #3",
+      "104":"d #3",
+      "105":"d #3",
```

Comment:
These are missing an `n`, should be `n d #3` :)

Reply:
whoops, one would think I should know that by now 🙃
thanks, fixed! :)